universal-application-tool-0.0.1/conf/evolutions/default/3.sql

```diff
@@ -11,7 +11,13 @@ create table if not exists questions (
   id bigserial primary key,
   object jsonb not null,
   -- ensure that all entries include a 'target' key.
-  check (object ? 'target')
+  check (
+    (object->'target') is not null
+      and
+    jsonb_typeof(object->'target') = 'string'
+      and
+    LENGTH((object->>'target')::varchar) > 0
+  )
```

Comment:
Suggestion (non-blocking): the latest design doesn't call for a `target` string or equivalent by a different name. Suggest removing this check.

Reply:
Sure, removed.